.github/workflows/tests.yml

```diff
@@ -24,7 +24,7 @@ defaults:
 
 jobs:
   tests:
-    name: Tests - ${{ matrix.runtime-version }} ${{ matrix.os }} py${{ matrix.python-version }}
+    name: Tests - ${{ matrix.os }} py${{ matrix.python-version }}
```

Comment:
Could you elaborate more on what value you see on continuing to test a latest and upstream build? Given our release cadence, I imagine we'll be updating the definition of latest roughly every two weeks without thinking much about it

Reply:
The benefit is that if somebody merges into main a performance regression, with `upstream` the CI bot will open a ticket for us the next morning, whereas without somebody will need to remember to manually update the environment here. In case of perceivedly uneventful releases, this may potentially not happen for a few months.

Reply:
Ah, I see. Sorry, it may not be clear based on the current state of this PR. I'd like to only have an `upstream` build that tests the latest `main` of `dask` / `distributed`

Reply:
Ah, I see. I'm happy to drop `latest`.

Reply:
👍 

This PR has gotten stale. I'll bring it back to life either later today or tomorrow